build.sbt

```diff
@@ -82,6 +82,17 @@ lazy val commonSettings = Release.publishSettings ++ Seq (
   )
 )
 
+lazy val sunriseModuleDependencies = Seq (
```

Comment:
this is duplicated, we could just have directly in build.sbt and it would be applied to all modules:

```scala
resolvers in ThisBuild ++= Seq (
     Resolver.sonatypeRepo("releases"),
     Resolver.sonatypeRepo("snapshots"),
     Resolver.mavenLocal
   )
```